garage/tf/q_functions/base.py

```diff
@@ -1,5 +1,25 @@
+import tensorflow as tf
+
 from garage.tf.core import Parameterized
 
 
 class QFunction(Parameterized):
-    pass
+    def _build_net(self, name, input_var):
```

Comment:
Making a private function part of a public interface is a strange choice. We will have to revisit this as we define TensorFlow models.